src/coreclr/jit/codegen.h

```diff
@@ -425,11 +425,9 @@ class CodeGen final : public CodeGenInterface
     void genOSRSaveRemainingCalleeSavedRegisters();
 #endif // TARGET_AMD64
 
-#if defined(TARGET_RISCV64)
-    void genStackProbe(ssize_t frameSize, regNumber rOffset, regNumber rLimit, regNumber rPageSize);
-#endif
-
+#if !defined(TARGET_RISCV64)
     void genAllocLclFrame(unsigned frameSize, regNumber initReg, bool* pInitRegZeroed, regMaskTP maskArgRegsLiveIn);
+#endif // TARGET_RISCV64
```

Comment:
```suggestion
    void genAllocLclFrame(unsigned frameSize, regNumber initReg, bool* pInitRegZeroed, regMaskTP maskArgRegsLiveIn);
```
No harm in having the unnecessary declaration.